app/models/institution.rb

```diff
@@ -221,17 +221,22 @@ def self.autocomplete(search_term, limit = 6)
   scope :filter, lambda { |field, value|
     return if value.blank?
     raise ArgumentError, 'Field name is required' if field.blank?
-    case value
-    when 'true', 'yes'
-      where(field => true)
-    when 'false', 'no'
-      where.not(field => true)
-    when 'school' # field: institution_type_name
-      where.not(field => EMPLOYER)
-    when 'employer' # field: institution_type_name
-      where(field => EMPLOYER)
+    if field == :category
```

Comment:
category is the only real "synthetic" query parameter that doesn't map straight through to a database value, so for clarity's sake I pulled it out of the single case statement that was here before.